Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -92,74 +127,91 @@ final class CategoryContentsViewController: BaseViewController {
     
     override func render() {
         
-        // 메인 뷰
-        self.view.addSubview(mainView)
-
-        // 필터 뷰
-        self.view.addSubview(filterView)
-        filterView.addSubview(totalLabel)
-        filterView.addSubview(gridButton)
-        filterView.addSubview(sortButton)
-        
-        // 필터 컬렉션 뷰
-        filterView.addSubview(filterCollectionView)
-        
-        // 메인 컨텐츠 뷰 
-        mainView.addSubview(contentsCollectionView)
+        self.view.addSubViews([mainView, filterView])
+        mainView.addSubViews([mainViewBorderView, contentsCollectionView])
+        filterView.addSubViews([totalLabel, gridButton, sortButton, filterCollectionView])
         
         mainView.snp.makeConstraints {
             $0.leading.bottom.trailing.top.equalTo(view.safeAreaLayoutGuide)
         }
-
+        
         filterView.snp.makeConstraints {
             $0.leading.trailing.equalTo(mainView)
             $0.top.equalTo(mainView).offset(17)
             $0.height.equalTo(67)
         }
-
+        
+        mainViewBorderView.snp.makeConstraints {
+            $0.leading.trailing.equalTo(mainView)
+            $0.top.equalTo(filterView.snp.bottom)
+            $0.height.equalTo(1)
+        }
+        
         totalLabel.snp.makeConstraints {
             $0.leading.equalTo(filterView).offset(16)
             $0.top.equalTo(filterView)
         }
-
+        
         gridButton.snp.makeConstraints {
             $0.top.equalTo(filterView)
-            $0.trailing.equalTo(filterView).offset(-16)
+            $0.trailing.equalTo(filterView).inset(16)
             $0.width.equalTo(18)
             $0.height.equalTo(18)
```

Comment:
p1;

```suggestion
            $0.width.height.equalTo(18)
```
이렇게도 가능합니다!

Reply:
3e0c780feff586bc41